src/App.js

```diff
@@ -1,26 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calc from './components/Calculator';
 
 function App() {
   return (
```

Comment:
- [ ] Kindly make your App.js component a class-based component. It's a requirement that these components be class-based components. Especially the Calculator and App components should be class-based for now 👍